src/Microsoft.Extensions.Configuration.UserSecrets/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Value cannot be null or an empty string.</value>
   </data>
   <data name="Error_EntryAssemblyNull" xml:space="preserve">
-    <value>This method could not find a user secret ID because the application's entry assembly is not set. Try using the ".AddUserSecrets(Assembly assembly)" method instead.</value>
+    <value>This method could not find a user secret ID because the application's entry assembly is not set. Try using the ".AddUserSecrets&lt;T&gt;()" or ".AddUserSecret(string userSecretsId)" method instead.</value>
```

Comment:
Oh actually here it's pretending like it knows you're using extension method syntax, so maybe less important. But my comment in email still applies, though it might not matter given my comment above about using `cref`.